ds_judgements_public_ui/templates/judgment/results.html

```diff
@@ -10,14 +10,9 @@ <h1>{% translate "results.search.title" %}</h1>
     <div class="results__container">
       <div class="results__list-layout-container">
         {% if context.show_no_exact_ncn_warning %}
-          <div class="tna-message">
-            <h class="tna-message__heading">
-            <span class="tna-message__heading-icon">I</span>mportant information
-            </h>
-            <div class="tna-message__message">
-              There is no judgment with the Neutral Citation of {{ context.query }} in our database.
-            </div>
-          </div>
+          {% with message="There is no judgment with the Neutral Citation of %s in our database."|interpolate:context.query %}
```

Comment:
I dislike this crafting of an interpolate function profoundly but reluctantly admit that it seems to be an established way of doing things.  (There is an `|add:`built-in, but apparently `"2"|add:"2"` is `"4"`)

Reply:
Yeah, i'm not a fan either tbh, but it seems like the 'django way' (`add` i considered but it doesn't let us interpolate the NCN into the middle of the string, as well as being confusingly overloaded on strings of numbers)